<?xml version="1.0"?>

<launch>
    <arg name="ns"        default="stereo_convert" />

    <node name="mini_ros_wrapper_node" pkg="mipi_ros_wrapper" type="mini_ros_wrapper_node" output="screen">
        <param name="left_image_raw_topic"    value="$(arg ns)/left/image_raw" />
        <param name="right_image_raw_topic"   value="$(arg ns)/right/image_raw" />
        <param name="left_frame_id"           value="$(arg ns)_left_frame" />
        <param name="right_frame_id"          value="$(arg ns)_right_frame" />
    </node>
</launch>
